shared/domains/chelonia/utils.js

```diff
@@ -498,12 +485,13 @@ export const recreateEvent = async (entry: GIMessage, rootState: Object): Promis
 
   if (!newRawOpV) return
 
+  const newOp = opT === GIMessage.OP_ATOMIC && (newRawOpV: any).length === 1
+    ? (newRawOpV: any)[0]
+    : [opT, newRawOpV]
```

Comment:
What's going on here with `OP_ATOMIC`? It seems like if it's an atomic op with 1 operation it's for some reason modifying the message that's being sent to remove the outer `OP_ATOMIC` wrapper... but that's wrong. Why is it modifying what the user wants?